packages/sanity/src/core/studio/components/navbar/search/contexts/search/reducer.ts

```diff
@@ -597,7 +597,11 @@ function stripRecent(terms: RecentSearch | SearchTerms) {
  *
  * Note that should any result appear again in subsequent pages, its first instance will be removed.
  */
-function deduplicate(hits: SearchHit[]): SearchHit[] {
+function deduplicate(hits: SearchHit[], {strategy}: {strategy?: SearchStrategy}): SearchHit[] {
+  if (strategy !== 'textSearch') {
```

Comment:
This seems to be changing the previous `textSearch` behavior right? 
Could we add some details on why we don't want to filter duplicates when using this strategy ?

Reply:
No, it shouldn't be. The intention is to skip deduplication for any strategy **that isn't** `textSearch`, so it returns the provided `hits` without processing them in that case.

The deduplication is only necessary when `textSearch` is enabled, because we found sometimes results are duplicated at the beginning and end of pages.

Reply:
You are right, sorry, miss read it as `===` 
I think adding this for our future selfs will be great 
>The deduplication is only necessary when `textSearch` is enabled, because we found sometimes results are duplicated at the beginning and end of pages.

Reply:
It could be more explicit, but it is already mentioned 1 line up in the comment:

 > At page boundaries, **the Text Search API may sometimes produce duplicate results**.

Reply:
Right, github was not showing me that piece. No need to add anything else